contrib/endpoints/src/api_manager/context/request_context.cc

```diff
@@ -130,13 +130,14 @@ RequestContext::RequestContext(std::shared_ptr<ServiceContext> service_context,
 
 std::string RequestContext::GetRequestHTTPMethodWithOverride() {
   std::string method;
-  request_->FindHeader(kHttpMethodOverrideHeader, &method);
+  bool is_override_present = request_->FindHeader(kHttpMethodOverrideHeader,
```

Comment:
True. But it is also not a good practice to have if statements with side effects as it is the case here. I think it is just a choice of having another variable or not. It should not affect the overall readability or performance. Let me know if you strongly feel about this...